ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12-periodics.yaml

```diff
@@ -31766,6 +31667,182 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build05
+  cron: 15 22 9,16,23,30 * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: release-4.12
+    org: openshift
+    repo: openshift-tests-private
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-qe
+    ci-operator.openshift.io/variant: multi-nightly
+    ci.openshift.io/generator: prowgen
+    job-release: "4.12"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-openshift-tests-private-release-4.12-multi-nightly-aws-ipi-amd-f7-netobserv
```

Comment:
The old job has reporter_config for slack notification, while the new jobs does not set that. Is it expected or you'd like to add it, too ?

Reply:
yeah it would be good to have it. But I dont see where it was added in the older job to add to the new job. I just copied the whole job from `amd` to `multi`

Reply:
For slack notification, it's added manually in the xxx-release-4.yy-periodics.yaml

Reply:
Yup added those manually, thanks for the info